src/parser/jobs/pld/modules/FightOrFlight.tsx

```diff
@@ -60,18 +62,42 @@ export class FightOrFlight extends BuffWindow {
 
 		this.addEvaluator(new ExpectedActionsEvaluator({
 			expectedActions: [
-				{action: this.data.actions.GORING_BLADE, expectedPerWindow: 1},
-				{action: this.data.actions.CONFITEOR, expectedPerWindow: 1},
-				{action: this.data.actions.BLADE_OF_FAITH, expectedPerWindow: 1},
-				{action: this.data.actions.BLADE_OF_TRUTH, expectedPerWindow: 1},
-				{action: this.data.actions.BLADE_OF_VALOR, expectedPerWindow: 1},
-				{action: this.data.actions.HOLY_SPIRIT, expectedPerWindow: 1},
+				{action: actions.GORING_BLADE, expectedPerWindow: 1},
+				{action: actions.CONFITEOR, expectedPerWindow: 1},
+				{action: actions.BLADE_OF_FAITH, expectedPerWindow: 1},
+				{action: actions.BLADE_OF_TRUTH, expectedPerWindow: 1},
+				{action: actions.BLADE_OF_VALOR, expectedPerWindow: 1},
 			],
-			suggestionIcon: this.data.actions.FIGHT_OR_FLIGHT.icon,
+			suggestionIcon: actions.FIGHT_OR_FLIGHT.icon,
+			suggestionContent: <Trans id="pld.fightorflight.suggestions.gcd_actions.content">
```

Comment:
_Action:_ It looks like you're using the same text (and translation key) in both suggestions; given how verbose and action-filled it is, I'd recommend paring them down and making each one unique and only about the actions the individual evaluator is tracking. For instance, this one could be shortened to:

`Try to land <DataLink action="GORING_BLADE" />, <DataLink action="CONFITEOR" />, <DataLink action="BLADE_OF_FAITH" />, <DataLink action="BLADE_OF_TRUTH" />, and <DataLink action="BLADE_OF_VALOR" /> during every <DataLink action="FIGHT_OR_FLIGHT" /> window.`

Or even simpler:

`Try to land your full <DataLink action="CONFITEOR" /> combo plus <DataLink action="BLADE_OF_VALOR" /> during every <DataLink action="FIGHT_OR_FLIGHT" /> window.`

Reply:
I've gone with your first suggestion. While the second one would also work, currently Blade of Honor is in the ogcd Evaluator, not the gcd evaluator. Therefore the shortened suggestion would feel incorrect without shifting Blade of Honor around to the gcd section.

I've also added the Blade of Honor suggestion text for the ogcd abilities.